src/Component/src/Doctrine/Persistence/Exception/ExistingResourceException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Doctrine\Persistence\Exception;
+
+class ExistingResourceException extends \Exception
```

Comment:
Is it good time to rename\adjust exception?

```suggestion
class ResourceExistsException extends \RuntimeException implements ExceptionInterface
```

```php
namespace Sylius\Resource\Doctrine\Persistence\Exception;

interface ExceptionInterface extends \Sylius\Resource\Exception\ExceptionInterface extends \Throwable
``` 

https://learn.microsoft.com/en-us/java/api/com.azure.core.exception.resourceexistsexception?view=azure-java-stable

Reply:
done, thanks a lot.